deepr/layers/losses.py

```diff
@@ -223,8 +219,8 @@ def forward(self, tensors, mode: str = None):
 
         losses = tf.add(true_losses, sampled_losses)
         # One loss per event, average of scores : (batch, num_events)
-        event_scores = WeightedAverage()((losses, tf.to_float(mask)))
+        event_scores = WeightedAverage()((losses, mask))
```

Comment:
ah, so you expect the mask to already be a float tensor?
it would maybe be safer to expect a bool, especially because we also need it

Reply:
yes, I do cast upper in the code